event.go

```diff
@@ -51,6 +51,7 @@ type Event struct {
 	Username            string               `json:"username,omitempty"`
 	WindowID            string               `json:"windowId,omitempty"`
 	WindowOptions       *WindowOptions       `json:"windowOptions,omitempty"`
+	Proxy               string               `json:"proxy,omitempty"`
```

Comment:
Use *[WindowProxyOptions](https://github.com/asticode/go-astilectron/blob/master/window.go#L159) instead of `string` here